datafusion/core/src/datasource/listing/mod.rs

```diff
@@ -78,7 +78,7 @@ pub struct PartitionedFile {
     ///
     /// DataFusion relies on these statistics for planning (in particular to sort file groups),
     /// so if they are incorrect, incorrect answers may result.
```

Comment:
its probably worthy mentioning in the comments the reason why  the `Option<Arc>` datatype was chosen